### ISSUE TITLE
Support for OSRM v5 API

### ISSUE THREAD

**Issue by noblige** [perliedman/leaflet-routing-machine#225]

**Comment by perliedman** [comment]
Work in progress: https://github.com/perliedman/leaflet-routing-machine/tree/osrm-5.0

**Comment by perliedman** [comment]
Closed by 4dc4844384b21f64707c346791ca2b93dc3f554b

**Comment by noblige** [comment]
@perliedman 'z' is no longer a parameter in v5. I was able to run frontend with this hash by removing 'z' parameter from URL.

```
diff --git a/src/L.Routing.OSRMv1.js b/src/L.Routing.OSRMv1.js
index 811704e..6a49de1 100644
--- a/src/L.Routing.OSRMv1.js
+++ b/src/L.Routing.OSRMv1.js
@@ -281,7 +281,7 @@
 
                        return this.options.serviceUrl + '/' + this.options.profile + '/' +
                                locs.join(';') + '?' +
-                               (options.z ? 'z=' + options.z + '&' : (options.geometryOnly ? 'overview=full&' : '')) +
+                               (options.geometryOnly ? 'overview=full&' : '') +
                                //'hints=' + hints.join(';') + '&' +
                                'alternatives=' + computeAlternative.toString() + '&' +
                                'steps=' + computeInstructions.toString() +
```
https://github.com/Project-OSRM/osrm-backend/wiki/New-Server-api
